### DISCUSSION TITLE
Configuration directory

### DISCUSSION THREAD

**Discussion by YoannCalamai** [sqlpage/SQLPage#596] [Q&A]
Hi,

I run into something strange. With Docker, it is like sqlpage 0.29 is looking for the config directory inside /var/www/sqlpage.
I may have missed something in the release notes.

I map my config directory with a bind mount volume to /etc/sqlpage and my src to /var/www. The app works but no migration is applied and my custom handlebars are not found.

I tried to change the env var SQLPAGE_CONFIGURATION_DIRECTORY and CONFIGURATION_DIRECTORY. My container has the right value inside these. But it change nothing.

It works on v0.28.0 but not with the image lovasoa/sqlpage:main

Here is the error message I got:
`sqlpage::webserver::database::migrations] Not applying database migrations because '/var/www/sqlpage/migrations' does not exist`

Here is a small example to replicate the issue
[configdir_issue.zip](https://github.com/user-attachments/files/17025298/configdir_issue.zip)

Can you help me, please?

**Comment by lovasoa** [comment]
I can confirm the bug. Thank you for the report ! I I'll look into it.

**Comment by lovasoa** [comment]
@YoannCalamai , I fixed it and added tests to prevent such regressions in the future. Thanks again for quickly noticing and prompting me :+1:

**Comment by YoannCalamai** [comment]
You welcome! Thanks for your work ;-)
